collect_app/src/main/java/org/odk/collect/android/tasks/SaveToDiskTask.java

```diff
@@ -232,11 +232,19 @@ private void updateInstanceDatabase(boolean incomplete, boolean canEditAfterComp
     }
 
     /**
-     * Return the name of the savepoint file for a given instance.
+     * Return the savepoint file for a given instance.
+     */
+    public static File getSavepointFile(String instanceName) {
```

Comment:
Is it totally necessary for this to be exposed?

Reply:
Just for testing